Azure/Azure-Sentinel
# Add FalconFriday 0xFF15 - 0xFF1B

Required items, please complete
   
   Change(s):
   - See guidance below

   Reason for Change(s):
   - See guidance below

   Version Updated:
   - Required only for Detections/Analytic Rule templates
   - See guidance below

   Testing Completed:
   - See guidance below

   Checked that the validations are passing and have addressed any issues that are present:
   - See guidance below
  
   Change(s):
   - Add newly released FalconFriday content

   Reason for Change(s):
   - Newly released FalconFriday content

   Version updated:
   - Yes

   Testing Completed:
   - Yes

   Checked that the validations are passing and have addressed any issues that are present:
   - Yes
